### DISCUSSION TITLE
Cannot edit link in workflow task Send Email

### DISCUSSION THREAD

**Discussion by Marshal4582** [YetiForceCompany/YetiForceCRM#17154] [How to]
Hi

Cannot enter link url in dialog of workflow task Send Email. Dialog seems to be partially inactive and console says:
Duplicate form field id in the same form
Multiple form field elements in the same form have the same id attribute value. This might prevent the browser from correctly autofilling the form.
To fix this issue, use unique id attribute values for each form field.

Bug or config?

**Comment by rskrzypczak** [comment]
What browser are you using?
..and if you can, please post a screenshot of a specific place in the system along with the error message.

**Comment by Marshal4582** [comment]
Chrome, Firefox, Opera - the result is the same
In shown below gif, I cannot type in Display Text and URL fields
Server config is perfect with latest stable 6.4. Logs blank except mentioned console msg.

![workflows-link](https://github.com/YetiForceCompany/YetiForceCRM/assets/50770738/d848cff0-f329-4619-8aff-73aa12b6caff)

**Reply by Marshal4582** [reply]
By the way, the same I can observe in YF Dev demo. If on your side things are correct it could be some config here, but without logs I feel lost
